app/models/practice.rb

```diff
@@ -50,9 +51,19 @@ def reset_searchable_practices
     clear_searchable_cache if self.reset_searchable_cache
   end
 
-  def self.searchable_practices
-    Rails.cache.fetch('searchable_practices') do
-      Practice.get_with_categories
+  def self.searchable_practices(sort = 'a_to_z')
```

Comment:
⚠️ Not sure if it is better to cache these sorted results leveraging ActiveRecord or just get the result of `Practice.get_with_categories_and_adoptions_ct` and then sort the results with ruby?? Let me know what you all think 🤔